README.md

```diff
@@ -90,5 +90,7 @@ See the [LICENSE file](LICENSE.txt) for license rights and limitations.
 
 ## Contributing
 
+[![Small Image](https://img.shields.io/badge/Contribute%20with-Gitpod-908a85?logo=gitpod)](https://gitpod.io/#https://github.com/mattermost/mattermost-server)
```

Comment:
Since the issue https://github.com/mattermost/mattermost-developer-documentation/issues/1243 was created, we have renamed this repo from `mattermost-server` to `mattermost`. The current state of this PR works because GitHub keeps a redirect from the old name to the new name. Although this works, I think we should have the updated name in this link for readability. What do you think @aniketh-varma?

```suggestion
[![Small Image](https://img.shields.io/badge/Contribute%20with-Gitpod-908a85?logo=gitpod)](https://gitpod.io/#https://github.com/mattermost/mattermost)
```